neurodamus/connection_manager.py

```diff
@@ -328,8 +334,8 @@ def _open_synapse_file(self, synapse_file, pop_name):
 
     def _init_conn_population(self, src_pop_name, pop_id_override):
         if not src_pop_name:
-            src_pop_name = self._src_cell_manager.population_name
-        dst_pop_name = self._cell_manager.population_name
+            src_pop_name = self.src_node_population
+        dst_pop_name = self.dst_node_population
```

Comment:
There are also other places that use `conn_manager.src_cell_manager.population_name`, i.e. cell_distribution.py, gap_junction.py. We can also replace there.

Reply:
I fixed a few, but in other places we already have the node_manager, so we directly access the property instead of going all the way from the conn_manager